### PR TITLE
Add method info support

### PR DESCRIPTION
**Pull request by dlandi**

### CODE REVIEW

File: Standard.Reflection/Models/Foundations/Methods/Exceptions/FailedMethodServiceException.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+using System;
+using Xeptions;
+
+namespace Standard.Reflection.Models.Foundations.Methods.Exceptions
+{
+    public class FailedMethodServiceException : Xeption
+    {
+        public FailedMethodServiceException(Exception innerException)
+        : base(message: "Failed Method Service Exception occurred, please contact support for assistance.", innerException)
```
Comment (cjdutoit) [inline Standard.Reflection/Models/Foundations/Methods/Exceptions/FailedMethodServiceException.cs:13]: Indent by one

File: Standard.Reflection/Services/Foundations/Methods/MethodService.Exceptions.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        private MethodInfo[] TryCatch(ReturningMethodsFunction returningMethodsFunction)
+        {
+            try
+            {
+                return returningMethodsFunction();
+            }
+            catch (Exception exception)
+            {
+                var failedMethodServiceException = new FailedMethodServiceException(exception);
+
```
Comment (cjdutoit) [inline Standard.Reflection/Services/Foundations/Methods/MethodService.Exceptions.cs:24]: Remove the empty line between the var's  or wrap the line below if you want it separated for readability

File: Standard.Reflection.Unit.Tests/Services/Foundations/Methods/MethodServiceTests.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            MethodInfo[] methods = Enumerable.Range(start: 0, count: randomMethodCount)
+                .Select(i => new Mock<MethodInfo>().Object)
+                    .ToArray();
+
+            return methods;
+        }
+
+        private static int GetRandomNumber()
+        {
+
```
Comment (cjdutoit) [inline Standard.Reflection.Unit.Tests/Services/Foundations/Methods/MethodServiceTests.cs:39]: Remove this line and add it between the line below and the return

File: Standard.Reflection.Unit.Tests/Services/Foundations/Properties/PropertyServiceTests.cs [hunk 4]
```diff
@@ -33,7 +34,11 @@ private static PropertyInfo[] CreateRandomProperties()
             return properties;
         }
 
-        private static int GetRandomNumber() =>
-            Random.Shared.Next(minValue: 3, maxValue: 8);
+        private static int GetRandomNumber()
+        {
+
```
Comment (cjdutoit) [inline Standard.Reflection.Unit.Tests/Services/Foundations/Properties/PropertyServiceTests.cs:39]: Remove this line and add it between the line below and the return

**Review by cjdutoit — CHANGES_REQUESTED** [review]
Some code cleanup tasks

**Comment by dlandi** [comment]
I have superseded this PR with another one after it so I will retire this one.

**Pull request closed without merging**